nyxtools/flyer-eiger2.py

```diff
@@ -16,37 +16,15 @@
 DEFAULT_DATUM_DICT = {"data": None, "omega": None}
 
 
-class NYXFlyer(MXFlyer):
+class NYXEiger2Flyer(MXFlyer):
     def __init__(self, vector, zebra, detector=None) -> None:
-        self.name = "NYXFlyer"
-        self.vector = vector
-        self.zebra = zebra
-        self.detector = detector
-
+        self.super().__init__(vector, zebra, detector)
+        self.name = "NYXEiger2Flyer"
         self.data_directory_name = None
-        self.file_prefix = None
-        self.num_images = None
-        self.file_number_start = None
-
-        self._asset_docs_cache = deque()
         self._resource_document = None
```

Comment:
this shouldn't be necessary at all - it's only used in the Pilatus-specific collect_asset_docs()

Reply:
I will be sure to get rid of that one.